molecule/cookiecutter/scenario/driver/hetznercloud/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/prepare.yml

```
@@ -0,0 +1,11 @@
+---
+{% raw -%}
+- name: Prepare
+  hosts: all
+  gather_facts: false
+  tasks:
+    - name: Install python for Ansible
+      raw: test -e /usr/bin/python || (apt -y update && apt install -y python-minimal python-zipstream)
```

Maybe we could multiline this?
```suggestion
      raw: >-
        test -e /usr/bin/python ||
        (apt -y update && apt install -y python-minimal python-zipstream)
```

---

Unrelated since this is a copy/pasta and all other files do that. I'm not refactoring here.